app/subscriber/src/features/my-folders/MyFolders.tsx

```diff
@@ -40,8 +40,8 @@ export const MyFolders = () => {
 
   const handleSave = () => {
     if (!!active) {
-      updateFolder(active).then(() => {
-        toast.success(`${active.name} updated successfully`);
+      updateFolder(active).then((data) => {
+        toast.success(`${data.name} updated successfully`);
         setMyFolders([...myFolders.filter((folder) => folder.id !== active.id), active]);
```

Comment:
This is the line that needs `data`.  I have a meeting for the next hour, but I can go over this with you later if you want?